### PR TITLE
Feature/automation app center apk upload

### PR DESCRIPTION
**Pull request by IvanYuliyanov**
Fixes #
Created new CI workflow to automatically upload .apk file in App Center.

To test:
Upload .apk file to the App Center after push to 'app-center' branch.

PR submission checklist:

- [x] I have followed the [Contributing Guidelines](https://github.com/scalefocus/virusafe-android/blob/master/CONTRIBUTING.md)
- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes and have described them above.

### CODE REVIEW

File: .github/workflows/app-center-apk-upload-ci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    - name: upload artefact to App Center group 2
+      uses: zackify/AppCenter-Github-Action@1.0.0
+      with:
+        appName: ${{ secrets.APP_CENTER_OWNER }}/${{ secrets.APP_NAME_DEV }}
+        token: ${{ secrets.APP_CENTER_TOKEN }}
+        group: ${{ secrets.APP_CENTER_GROUP_2 }}
+        file: presentation/build/outputs/apk/debug/presentation-debug.apk
+        
+    - name: Test
```
Comment (s-yanev) [inline .github/workflows/app-center-apk-upload-ci.yml]: I think we can remove this 'Test' stage.

File: build.gradle [hunk 2]
```diff
@@ -5,8 +5,8 @@ buildscript {
         minSdkVersion = 21
         targetSdkVersion = 29
 
-        versionCode = System.getenv("BUILD_NUMBER") as Integer ?: 1
-        versionName = "1.0.0"
+        versionCode = System.getenv("GITHUB_RUN_NUMBER") as Integer ?: 26
+        versionName = "1.0.23"
```
Comment (s-yanev) [inline build.gradle]: Lets leave the versionCode = 1 and versionName = 1.0.0 as a defoult values and apply the GITHUB_RUN_NUMBER also on the versionName property. Something like "1.0.${System.getenv("GITHUB_RUN_NUMBER")}"

File: .github/workflows/app-center-apk-upload-ci.yml (hunk 1 above)
Comment (IvanYuliyanov) [reply .github/workflows/app-center-apk-upload-ci.yml]: Done.

File: build.gradle (hunk 2 above)
Comment (IvanYuliyanov) [reply build.gradle]: Done.

**Review by s-yanev — APPROVED** [review]

**Pull request merged**